portal/gil/templates/gil/macros.html

```diff
@@ -5,17 +5,22 @@
         </div>
         <ul class="nav-list">
           <li class="nav-list__item"><a href="{{url_for('gil.landing')}}">{{_("Home")}}</a></li>
+          <li class="nav-list__item"><a href="{{url_for('gil.prostate_cancer_facts')}}">{{_("Prostate Cancer Facts")}}</a></li>
           <li class="nav-list__item"><a href="{{url_for('gil.about')}}">{{_("About")}}</a></li>
+          <!--
```

Comment:
I assume this will just comment these nav elements out, why not remove the nav element HTML entirely?